.travis.yml

```diff
@@ -34,6 +34,12 @@ jobs:
         - pip install -r requirements.txt
         - sls plugin install -n serverless-python-requirements
         - sls deploy
+        - pip install awscli
+        - npm install -g @angular/cli
```

Comment:
I think that we should put all node dependencies in the `package.json`, then a only single `npm install` will be necessary.

Reply:
Only problem is that ng doesn't get added to the path that way...might be a way around that.